utilities/transactions/transaction_test.cc

```diff
@@ -543,13 +579,16 @@ TEST_P(TransactionTest, SharedLocks) {
   ASSERT_TRUE(txn3);
 
   // Test shared access between txns
-  s = txn1->GetForUpdate(read_options, "foo", nullptr, false /* exclusive */);
+  s = txn1->GetForUpdate(read_options, "foo", (std::string*)nullptr,
```

Comment:
C-style cast is not recommended but C++ doesn't offer any good alternatives that aren't verbose or still involve a C-style cast:

```
static_cast<std::string*>(nullptr)
std::add_pointer_t<std::string>{}
(std::string*){}
```

https://google.github.io/styleguide/cppguide.html#Casting